manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -82,22 +82,50 @@ spec:
     rules:
     - alert: UsingDeprecatedAPIAppsV1Beta1
       annotations:
-        message: A client in the cluster is using deprecated apps/v1beta1 API that will be removed soon.
+        summary: |
+          A client is using a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version
+        message: |
+          A client in the cluster has used a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version of the {{"{{"}} $labels.resource {{"}}"}} resource in the last 24 hours.
+          This API version will be removed soon; please migrate to the current version of this API.
       expr: |
-        apiserver_request_count{group="apps",version="v1beta1"}
+        increase(
+          apiserver_request_total{
+            group="apps",
+            version="v1beta1",
+          }[24h]
+        ) > 0
       labels:
         severity: warning
     - alert: UsingDeprecatedAPIAppsV1Beta2
       annotations:
-        message: A client in the cluster is using deprecated apps/v1beta2 API that will be removed soon.
+        summary: |
+          A client is using a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version
+        message: |
+          A client in the cluster has used a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version of the {{"{{"}} $labels.resource {{"}}"}} resource in the last 24 hours.
+          This API version will be removed soon; please migrate to the current version of this API.
       expr: |
-        apiserver_request_count{group="apps",version="v1beta2"}
+        increase(
+          apiserver_request_total{
+            group="apps",
+            version="v1beta2",
+          }[24h]
+        ) > 0
       labels:
         severity: warning
     - alert: UsingDeprecatedAPIExtensionsV1Beta1
       annotations:
-        message: A client in the cluster is using deprecated extensions/v1beta1 API that will be removed soon.
+        summary:
+          A client is using a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version
+        message: |
+          A client in the cluster has used a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version of the {{"{{"}} $labels.resource {{"}}"}} resource in the last 24 hours.
+          This API version will be removed soon; please migrate to the current version of this API.
       expr: |
-        apiserver_request_count{group="extensions",version="v1beta1",resource!~"ingresses|",client!~"hyperkube/.*|cluster-policy-controller/.*"}
+        increase(
+          apiserver_request_total{
+            group="extensions",
+            version="v1beta1",
+            client!~"hyperkube/.*|cluster-policy-controller/.*|velero-server/.*"
```

Comment:
We need to add also:
openshift-controller-manager and kube-state-metrics to this list, as we have that firing in our cluster.